<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainShell</class>
 <widget class="QWidget" name="MainShell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vibe Snitch - AI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(
        spread:pad,
        x1:0, y1:0,
        x2:1, y2:1,
        stop:0 #1c192b,
        stop:0.2 #434262,
        stop:0.4 #25223c,
        stop:0.6 #363353,
        stop:1 #151321
    );
}</string>
  </property>
  <widget class="QFrame" name="headerFrame">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>960</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>960</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>960</width>
     <height>80</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#headerFrame {
background-color: #1a1328;
	border: 2px solid#26183d;
	border-radius: 20px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="iconUser">
      <property name="maximumSize">
       <size>
        <width>33</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>34</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #dbd2d4;
background-color: #1a1328;
margin-left: 3px;
font-size: 34pt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../assets/user.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblUsername">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #dbd2d4;
background-color: #1a1328;
margin-bottom: 3px;
</string>
      </property>
      <property name="text">
       <string>Ayaan Ahmed</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3a1a6e;
color: #dbd2d4;
border-radius: 15px;
padding: 12px;

}

QPushButton:hover {
    background-color: #892CDC;
}</string>
      </property>
      <property name="text">
       <string> Home</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../assets/home.png</normaloff>../assets/home.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3a1a6e;
color: #dbd2d4;
border-radius: 15px;
padding: 12px;

}

QPushButton:hover {
    background-color: #892CDC;
}</string>
      </property>
      <property name="text">
       <string> Saved Results</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../assets/rectangle-vertical-history.png</normaloff>../assets/rectangle-vertical-history.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>140</y>
     <width>965</width>
     <height>545</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>965</width>
     <height>545</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>965</width>
     <height>545</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QStackedWidget#stackedWidget {
background-color: #1a1328;
	border: 2px solid#26183d;
	border-radius: 20px;
}


</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
